packages/ethash/test/miner.spec.ts

```diff
@@ -6,11 +6,11 @@ import * as tape from 'tape'
 import { Ethash } from '../src'
 
 import type { BlockHeader } from '@ethereumjs/block'
-
+const cacheDb = new MemoryLevel()
 const common = new Common({ chain: Chain.Ropsten, hardfork: Hardfork.Petersburg })
 
 tape('Check if miner works as expected', async function (t) {
-  const e = new Ethash(new MemoryLevel())
+  const e = new Ethash(cacheDb as any)
```

Comment:
wouldn't it be good to alloc new db each fime? may be in beforeeach?

Reply:
That's a matter of opinion I guess.  The above change just has this code match the original test structure (which reused the same DB each time).